pd3/src/main/java/de/dema/pd3/persistence/ChatroomUser.java

```diff
@@ -0,0 +1,67 @@
+package de.dema.pd3.persistence;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+
+@Entity
+public class ChatroomUser implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+
+	@EmbeddedId
+	private ChatroomUserId id;
```

Comment:
Imho kannst du auf beides verzichten, also auf EmbeddedId und auf @IdClass (siehe: https://hellokoding.com/jpa-many-to-many-extra-columns-relationship-mapping-example-with-spring-boot-maven-and-mysql/).
Oder einfach aus dem anderen Branch kopieren, dort hatte ich es auch ohne hinbekommen (glaub ich :).
Ich nehme an, Hibernate schnallt anhand der OneToMany(+mapping) annotation wie es zusammengehört.